operators/pkg/instctrl/containers.go

```diff
@@ -28,6 +28,27 @@ import (
 	"github.com/netgroup-polito/CrownLabs/operators/pkg/utils"
 )
 
+const (
+	NFSVolumeName = "nfs-vol"
+)
+
+func (r *InstanceReconciler) GetNFSVolume(ctx context.Context, volume *v1.Volume, volumeMount *v1.VolumeMount) error {
```

Comment:
Seems unused to me, and could be dropped.

Reply:
Dropped